bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/handler/LinkyHandler.java

```diff
@@ -108,8 +107,8 @@ public LinkyHandler(Thing thing, LocaleProvider localeProvider, Gson gson, HttpC
         });
 
         this.cachedPowerData = new ExpiringDayCache<>("power cache", REFRESH_FIRST_HOUR_OF_DAY, () -> {
-            LocalDate to = LocalDate.now().plusDays(1);
-            LocalDate from = to.minusDays(2);
+            LocalDate to = LocalDate.now();
+            LocalDate from = to.minusDays(1);
```

Comment:
Are you sure it is not necessary to minus 2 days to be be certain that the previous day will be fully covered ?
I don't know if time is considered or only date.

Reply:
The request is only date based. When I made the test, keeping minusDays(2), I landed on the 31st of March.

Reply:
Ok